test/integration_test/specs/mysql-1-pvc-post-exec-rule-backup/mysql-post-backup-rule.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: stork.libopenstorage.org/v1alpha1
+kind: Rule
+metadata:
+  name: mysql-post-backup-rule
+  namespace: mysql-1-pvc-mysql
+  annotations:
+    "stork/cmdexecutor-image": "openstorage/cmdexecutor:latest"
+rules:
+  - podSelector:
+      app: mysql
+    actions:
+    - type: command
+      # dummy action to test non-background command
+      value: echo "hello stork pre-backup"
+      runInSinglePod: true
+    - type: command
+      runInSinglePod: true
+      # this command is just to test multiple background commands
+      value: mysql --user=root --password=$MYSQL_ROOT_PASSWORD -Bse 'show databases;system ${WAIT_CMD};'
+    - type: command
```

Comment:
Remove this command, shouldn't be a post-exec rule.

Reply:
Fixed.